src/App.js

```diff
@@ -0,0 +1,28 @@
+import { CapStatus } from './CapStatus';
+import { AllCapsules } from './AllCapsules';
+import   rocket  from './assets/rocket.svg'
+import styles from './App.module.css';
+
+function App() {
+  return (
+    <div className={styles.App}>
+      <header className={styles.Initial} >
+        <img className={styles.rocketLogo} src={rocket} alt="rocket logo"/>
+        <div className={styles.titles}>
+          <h1 className={styles.RocketTitle} >ROCKET</h1>
+          <h2 className={styles.RocketSubtitle} >IMPROVING ALL THE TIME</h2>
+        </div>
+        <button className={styles.startButton} type="button"> START </button>
+      </header>
```

Comment:
En ocaciones es bueno segmentar las responsabilidades tal vez esto pudo estar en otro archivo y solo invocarlo.
```suggestion
    <Header />
```

Reply:
Así como lo hiciste con <CapStatus /> & <AllCapsules />